### PR TITLE
Capstone project - Conference page

### PR DESCRIPTION
**Pull request by Nuri1977**
## Capstone project - Conference page: 

1. Linters and initial setup
2. Mobile view for Home page
3. Mobile view for About page
4. Desktop view for Home and About page
5. Added dropdown for header navigation
6. Dynamic rendering of Speakers section
7. Dynamic hide and show of speakers in mobile view
8. Updated read me file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,158 @@
...
+			<h3 class="more-courses">Join WTSQ summit 2022</h3>
+			<div class="div-program">
+				<h3 class="hole-program">SEE THE HOLE PROGRAM</h3>
+			</div>
+		</div>
+	</section>
+<!-- FEATURED SPEAKERS SECTIONS-->
+	<section id="intructors">
+		<div class="inner-istructors container">
+			<h3 class="instructors-title">Fetaured Speakers</h3>
```
Comment (leolpaz) [inline index.html]: - [Optional] There is a typo here, the project would look better if you fixed it

File: index.html [hunk 2]
```diff
@@ -1,15 +1,158 @@
...
+						<h4 >Lorem ipsum dolor sit amet.</h4>
+						<hr/>
+						<h5>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eum, quod.</h5>
+					</div>
+				</li>
+			</ul>
+			<button type="button" class="btn-more">More <i class="fas fa-chevron-down"></i></button>
+			<button type="button" class="btn-less">Show Less <i class="fas fa-chevron-up"></i></button>
+		</div>
+	</section>
```
Comment (leolpaz) [inline index.html]: - [ ] Please correct me if I'm wrong, but this piece of commented code doesn't serve any purpose (unlike the ones you used to improve readability like `<!-- PARTNERS SECTIONS-->` which are fine), and seem to have been used as a placeholder as you wrote the JS, kindly remove this, as pointed by HTML and CSS best practices

image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @Nuri1977 , 

You've done an amazing job so far, I'd like to highlight the following:
- Congratulations on creating the page
- Correct linter was used and it's passing

Some issues need to be resolved before your approval, but you're almost there!



## Required Changes ♻️

### - You're missing the video on your PR description, kindly add it
image



_Also check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nuri1977** [comment]
https://www.loom.com/share/980288102ea748f5b2b30c7b3539ca91

File: index.html (hunk 2 above)
Comment (Nuri1977) [reply index.html]: Thank you very much for your kind suggestions 😃 👍 .

**Review by julie-ify — APPROVED** [review]
## Hi @Nuri1977, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 
- Good video presentation ✔️ 
- Good job implementing the show more/ see less toggler 👍 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**